rlclientlib/vw_model/safe_vw.cc

```diff
@@ -120,6 +122,24 @@ void safe_vw::parse_context_with_pdf(string_view context, std::vector<int>& acti
   for (auto&& ex : examples) { _example_pool.emplace_back(ex); }
 }
 
+void safe_vw::add_lru_dedup_cache(uint64_t hash, std::string action_str)
+{
+  if (_dedup_cache == nullptr) { _dedup_cache = new lru_dedup_cache(); }
```

Comment:
is this something that will be passed into `safe_vw` or created by `safe_vw`? it should not be both, ownership should be clear

Reply:
removed this